src/elements/Table/Table.tsx

```diff
@@ -321,6 +370,8 @@ export class Table extends React.Component<IProps, any> {
                             })}
                         </tbody>
                     </table>
+                    {this.getPagination()}            
```

Comment:
Maybe instead of this line we can move the pagination into a separate component and then do `<Pagination data={this.props.data} total={this.props.total} pageSize={this.props.pageSize} />`